File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.java
class Solution {
    public int longestConsecutive(int[] nums) {
        Set<Integer> set=new HashSet<>();
        for(int i:nums){
            set.add(i);
        }
        int ans=0;
        for(int i : set){
            if (!set.contains(i-1)){
                int cur=i+1;
                while(set.contains(cur)){
                    cur+=1;
                }
                int lastel=cur-1;
                ans=Math.max(ans,lastel-i+1);
            }
        }
        return ans;
        
    }
}

File: Data Structures & Algorithms/maximum-subarray/submission-0.py
class Solution:
    def maxSubArray(self, nums: List[int]) -> int:
        cursum=0
        maxi=-(2**31)
        for i in nums:
            cursum+=i
            maxi=max(maxi,cursum)
            if cursum<0:
                cursum=0
        return maxi
        